webserver/templates/datasets/eval-job.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+
+{% block title %}Dataset "{{ dataset['name'] }}" by {{ author.musicbrainz_id }} - AcousticBrainz{% endblock %}
+
+{% block content %}
+  <div id="evaluation-results">
+
+    {% from 'datasets/macros.html' import print_header %}
+    {{ print_header(dataset, author, current_user, 'evaluation') }}
+
+    {% if job['status'] == 'pending' %}
```

Comment:
We can actually say "pending", maybe with a "time since added" option. In the future we could have an estimated ETA, but I don't think that's important. How about "n datasets in queue"?